Source/Turbo Navigator/TurboNavigator.swift

```diff
@@ -215,11 +215,18 @@ extension TurboNavigator: TurboNavigationHierarchyControllerDelegate {
         }
     }
 
-    func refresh(navigationStack: TurboNavigationHierarchyController.NavigationStackType) {
+    func refreshVisitable(navigationStack: TurboNavigationHierarchyController.NavigationStackType,
+                          newTopmostVisitable: any Visitable) {
         switch navigationStack {
-        case .main: session.reload()
```

Comment:
We can't use session.reload() because topmostVisitable does not change when a view controller is popped (or dismissed). For example:

- Propose visit to URL "/one"
- Visit completes, session.topmostVisitable = "one"
- Propose visit to URL "/two"
- Visit completes, session.topmostVisitable = "two"
- Propose visit to URL "/refresh_historical_location" with presentation: refresh
- Visitable with visitableURL = "/two" is popped.
- session.reload() is called, which reloads the topmost visitable.